source/develop/release-management/features/network/ipv6_subnets.html.md

```diff
@@ -56,6 +60,28 @@ The following configurations, which are explained in the [IPAM types](#ipam-type
 
 * static addresses
 * stateful DHCPv6
+* stateless DHCPv6
+
+### Stateless DHCPv6 / SLAAC motivation
+
+ISC DHCP client - dhclient - has up to now relied on an hardcoded prefix length
```

Comment:
afaik, 'a' vs 'an' article is tied to the sound of the word that follows - e.g. hardcoded. Since the 'h' is silent, it's as if it started with an 'a', which, begs for the 'an' article.

Are you sure it's an 'a' ?

Reply:
Yes

Reply:
I disagree, check [0].

[0] - https://www.englishpage.com/articles/a-vs-an.htm

[0] - https://www.englishpage.com/articles/a-vs-an.htm